dev/docker/hive/start.sh

```diff
@@ -4,15 +4,16 @@
 # This software is licensed under the Apache License version 2.
 #
 
-# start ssh
```

Comment:
@mchades  Do we need also remove `apt-get install openssh-server` from `hive/Dockerfile/?

Reply:
I'm not sure if removing this service will cause other issues, this requires more testing. 

I suggest keeping it for now and handling it in a separate PR if necessary.